api/label.go

```diff
@@ -19,6 +19,43 @@ const (
 	TenantLabelKey = "bb.tenant"
 )
 
+// LabelKeyRaw is the store model for an LabelKey.
+// Fields have exactly the same meanings as LabelKey.
+type LabelKeyRaw struct {
+	ID int
+
+	// Standard fields
+	CreatorID int
+	CreatedTs int64
+	UpdaterID int
+	UpdatedTs int64
+
+	// Domain specific fields
+	// bb.environment is a reserved key and identically mapped from environments. It has zero ID and its values are immutable.
+	Key       string
+	ValueList []string
+}
+
+// ToLabelKey creates an instance of LabelKey based on the LabelKeyRaw.
+// This is intended to be called when we need to compose an LabelKey relationship.
+func (raw *LabelKeyRaw) ToLabelKey() *LabelKey {
+	labelKey := LabelKey{
+		ID: raw.ID,
+
+		// Standard fields
+		CreatorID: raw.CreatorID,
+		CreatedTs: raw.CreatedTs,
+		UpdaterID: raw.UpdaterID,
+		UpdatedTs: raw.UpdatedTs,
+
+		// Domain specific fields
+		// bb.environment is a reserved key and identically mapped from environments. It has zero ID and its values are immutable.
+		Key: raw.Key,
+	}
+	labelKey.ValueList = append(labelKey.ValueList, raw.ValueList...)
```

Comment:
:-) never use copy

Reply:
@d-bytebase Could you please elaborate this?